terraform/domains/environment_domains/config/development.tfvars.json

```diff
@@ -0,0 +1,17 @@
+{
+  "hosted_zone": {
+    "schoolexperience.education.gov.uk": {
+      "front_door_name": "s189p01-gsedomains-fd",
+      "resource_group_name": "s189p01-gsedomains-rg",
+      "domains": [
+        "development"
+      ],
+      "cached_paths": [
+        "/packs/*"
+      ],
+      "environment_short": "dv",
+      "origin_hostname": "get-schools-experience-development.test.teacherservices.cloud",
```

Comment:
As discussed with @temitope777 : the name will be changed to get-school-experience... (no S)

Reply:
Updated